src/editor tweaks.js

```diff
@@ -0,0 +1,451 @@
+/*
+👣
+@file editor tweaks
+@summary Various adjustments to the editor that are too minor for individual plugins.  Each tweak is togglable.
+@license MIT
+@author Violgamba (Jon Heard)
+
+@description
+Various editor tweaks, or adjustments to the editor that are too minor for individual plugins.
+
+This plugin's event contains a togglable text field for each tweak.  The field defaults to "true",
+and can be set to "false" to disable the tweak.
+NOTE: togglable text fields were chosen over tag fields for easier toggling.
+
+Below are a list of the tweaks, starting with their togglable field.  Note that this list is subject
+to change as new tweaks are added to this plugin.
+
+TWEAKS LIST
+- tweak--new-plugins-have-filename-tag - Each imported plugin's fields will begin with a tag named
+	after the filename the plugin was loaded from.  This lets one quickly identify plugin events.
+- tweak--new-fields-default-to-tag-type - New fields begin as "tag" types, rather than "text" types.
+	This is useful for users who add tag events more frequently than text events.
+- tweak--tileset-export-ordered-by-tile - When the tileset is exported, the graphics are re-ordered
+	to match the tile order set in the editor.  This lets the user define the export order and makes
+	sure that all animation frames are consecutive.
+- tweak--save-button-shows-when-saved - When the game is saved, the save button dims until a change
+	occurs, wherein it turns back to yellow.  If the change is undone, the save button dims again.
+	This lets one know when the editor is in a saved state, removing the need for redundant save
+	clicks to be sure.  NOTE: the button can still be clicked while dimmed.
```

Comment:
i'd recommend submitting this to bipsi itself instead of as an editor plugin in here. whereas the tweaks above are pretty specific, this is a general UX improvement, and unless candle disagrees it would make more sense to share that with everyone instead of requiring plugins

Reply:
I think it'd be great if this became part of the main application, but...
I'm inclined to add this as a plugin, but also submit to bipsi.  When/if candle accepts this feature, the plugin can be removed from the listings.  Based on my work submitting the editor plugin support, I suspect it might be a while before something this involved makes it into the application.

Reply:
i'd still prefer at least posing the feature to candle and finding out if they want it/if it will take awhile before adding either of these here. if we're talking about a span of days then it's unlikely the delay would affect anyone but you (and you ofc already have local copies), and it would be a waste of time to get the plugin cleaned up for general use instead of just spending that time on the native feature imo

Reply:
Okeydokey.  I'm going to wait until he has looked at the pull request for the dynamic CONFIG for editor plugins (in order to to avoid spamming him with too many queries) then I'll ask him on discord.

Reply:
fyi candle's they/them